src/components/IconCustomizer.svelte

```
@@ -1,63 +1,41 @@
-	import { extensions as uiStateEx } from '../models/UIState';
-	import * as _selectedIcon from '../models/SelectedIcon';
+	import * as _selectedIcon from '../models/CustomizableIcon';
 	import { IconService } from '../services/icon.service';
-	import { selectedIcon } from '../stores/selected-icon.store';
-	import { uiState } from '../stores/ui-state.store';
-	import { UserIconService } from '../services/user-icon.service';
+	import { customizedIcon } from '../stores/icon-customizations.store';
 	import _ from 'lodash';
 
-	$: if ($selectedIcon) {
+	$: if ($customizedIcon?.userIconId && $customizedIcon.userIconId !== UUID.empty) {
```

ESLint

[error] Properties of objects and arrays are not reactive unless in runes mode. Changes to this property will not cause the reactive statement to update(reactive_declaration_non_reactive_property)

(svelte/valid-compile)